src/traits.md

```diff
@@ -25,7 +25,19 @@ impl Greet for Cat {
     }
 }
 
+// Generic function (more in the next chapter).
```

Comment:
Could you try enabling the "aspect-ratio helper" preprocessor here: https://github.com/google/comprehensive-rust/blob/main/book.toml#L24. I think you'll quickly see that there is not enough space on the page for such a long code example.